src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/SscsCaseData.java

```diff
@@ -219,6 +219,8 @@
     private String adjournCaseNextHearingDateOrTime;
     private String adjournCaseNextHearingFirstAvailableDateAfterDate;
     private String adjournCaseNextHearingFirstAvailableDateAfterPeriod;
+    private String hearingRoute;
```

Comment:
These two should be on the Hearings object, HearingDetails I believe. I'd also make the field an enum so we have a solid definition of what each type is.

I'd also probably call it something like HearingBookingType to make it clearer.